backend/internal/config/config.go

```diff
@@ -21,6 +21,8 @@ const (
 
 // Numbers corresponding to areas on the table, as received from node application
 var (
-	WhiteTeamArea = [4]int{20, 21, 23, 25}
-	BlueTeamArea  = [4]int{22, 24, 26, 27}
+	WhiteTeamArea     = [4]int{20, 21, 23, 25}
+	BlueTeamArea      = [4]int{22, 24, 26, 27}
+	BlueTeamGoalArea  = 27
+	WhiteTeamGoalArea = 20
```

Comment:
Maybe it would be a good idea to link somehow `BlueTeamGoalArea` with `BlueTeamArea` and `WhiteTeamGoalArea`  with `WhiteTeamArea` in case the team areas are changed in the future.

Reply:
Done